web/src/pages/Route/components/Step1/ProxyRewrite.tsx

```diff
@@ -154,88 +163,137 @@ const ProxyRewrite: React.FC<RouteModule.Step1PassProps> = ({ form, disabled })
     }
   };
 
-  return (
-    <PanelSection title={formatMessage({ id: 'page.route.panelSection.title.requestOverride' })}>
+  const SchemeComponent: React.FC = () => {
```

Comment:
more clearly, love this enhancement❤️

Reply:
thanks! It would be more clear than before!